CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Unreleased ()
+
+FEATURES:
+* **New resource** `hcp_hvn_route` (#122)
+
+⚠️ Note: Due to the introduction of HVN routes, this version introduces breaking changes to the `hcp_aws_network_peering` and `hcp_aws_transit_gateway_attachment` resources. Please see our [migration guide](link TODO) to learn about how to update your HCP networking setup to work with HVN routes. ⚠️
```

Comment:
do we a link already?

Reply:
Yeah I'm going to link to your PR with the migration steps here, since I don't think we'll keep the migration guide in the registry docs forever.